platform/android/java/app/config.gradle

```diff
@@ -28,6 +28,22 @@ ext.getExportPackageName = { ->
     return appId
 }
 
+ext.getExportVersionCode = { ->
+    String versionCode = project.hasProperty("version_code") ? project.property("version_code") : "" //1 is the default version code
+    if (versionCode == null || versionCode.isEmpty()) {
```

Comment:
Can you use ?: here?

Reply:
The function that this is based off of doesn't use ternary operators here. I think it's safer to follow [precedent](https://github.com/amanj120/godot/pull/17/files#diff-98ef5299ba59486356224f56e890e5f6R25)

Reply:
Sure, that's fine.